docs(update-tool): notify tool script renaming (#2968)
bonitasoft/bonita-doc: modules/ROOT/pages/release-notes.adoc

You can see the xref:setup-dev-environment:logging.adoc#logging-configuration[logging configuration] for more details.

== Notable changes

=== Update tool script has been renamed

To simplify and rationalize the naming of the scripts, the update tool script has been renamed.

The formerly named `bonita-update-tool.sh/.bat` script has been renamed to `bonita-update.sh/.bat`.

> Moreover, the Subscription version of the script is also named `bonita-update.sh/.bat`. (there is no `-sp` suffix anymore)

Reviewer: **suggestion:** 
```suggestion
In addition, the Subscription version of the script is also named `bonita-update.sh/.bat`. (there is no longer a `-sp` suffix).
```
